packages/grid/_modules_/grid/hooks/utils/useResizeContainer.ts

```diff
@@ -6,44 +6,36 @@ import { optionsSelector } from './optionsSelector';
 
 export function useResizeContainer(apiRef): (size: ElementSize) => void {
   const gridLogger = useLogger('useResizeContainer');
-  const widthTimeout = React.useRef<any>();
-  const heightTimeout = React.useRef<any>();
   const { autoHeight } = useGridSelector(apiRef, optionsSelector);
 
   const onResize = React.useCallback(
     (size: ElementSize) => {
-      clearTimeout(widthTimeout.current);
-      clearTimeout(heightTimeout.current);
+      // jsdom has no layout capabilities
+      const jsdom = /jsdom/.test(window.navigator.userAgent);
```

Comment:
isJsdom?

Reply:
We have used isX and x for naming boolean interchangeably in the codebase so far. How about we allow both, in the same way we allow both function and arrow functions?

Reply:
well it's not really the same. 
Adding isX or hasX removes any confusion of the type of the variable.
Some var such as rows / hasRows can be really confusing without the verb.

Reply:
Ok, I will use `isJSDOM` as we have a precedent in https://github.com/mui-org/material-ui/blob/f23207bf2312eba2686522f904d72aea7d3c0a88/packages/material-ui/src/Box/Box.test.js#L59